spec/models/miq_task_spec.rb

```diff
@@ -316,4 +316,45 @@
       expect(Job.count).to eq 0
     end
   end
+
+  describe "#attributes_hash_when_state_change" do
+    let(:miq_task) { FactoryGirl.create(:miq_task_plain) }
+    before(:each) do
+      @miq_task = FactoryGirl.create(:miq_task_plain)
```

Comment:
Why are you creating two miq_tasks (one via a let, and one via an instance var)?